wireapp/kalium
# feat: handle group settings receipt mode changed

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-2712 - Show a system message when read receipts are turned on](https://wearezeta.atlassian.net/browse/AR-2712)

This PR is only for handling receipt mode from changes from group settings screen(in-app and from other sources as events).

### Issues

There was no handling of showing a system message for when receipt mode was changed on a group setting.

### Causes (Optional)

It was not implemented.

### Solutions

Add implementation to always add a system message informing the members of the conversation that the receipt mode value was changed.

### Dependencies (Optional)

A PR in AR will be raised upon this one being merged

### Testing

#### How to Test

Run AR with this branch and change receipt mode in web client, then check database table `MessageConversationReceiptModeChangedContent` that is has entries.
